### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by connecteev** [tizis/laravel-comments#2]
Thanks for the package but the instructions in the readme are a bit hard to follow.

I created a new repo and tried to follow the instructions. I dont know what to do after this:
https://github.com/connecteev/test_tizis_laravel_comments

If you could point us to a sample repo that is using this package, that would be useful. Once again, thank you for creating this.

**Comment by iooe** [comment]
Hi.
https://github.com/tizis/laravel-comments-example


Setup .env, composer & npm and import test.sql to your local test database.

**Comment by iooe** [comment]
> the instructions in the readme are a bit hard to follow.


Sorry. English is my secondary language.

**Comment by connecteev** [comment]
@tizis thank you Roman. I will try this out today and let you know what I find. I appreciate the example repo!
